index.d.ts

```diff
@@ -19,8 +19,13 @@ export function trackPromise(promise: Promise <any> ): Promise <any>;
  * @returns It returns a new component that extends the input one.
  */
 
-export interface TrackerProps {
-  trackedPromiseInProgress
+export interface ComponentToWrapProps {
+  area: string;
```

Comment:
Should be here are optional?

Reply:
Or the other way around? the other hoc shouldn't be optional? not sure

Reply:
I see HOC is optional, because in common cases you want use areas, is something that you can ignore

Reply:
Exactly, in HOC is optional, and the HOC is always feeding the area to "component to wrap" (default area if it's undefined)